src/components/LogInModal/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Why is this necessary?

Reply:
Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.